benchmarks/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+if(NOT TARGET gtest OR NOT TARGET gmock)
+  # Prevent overriding the parent project's compiler/linker settings on Windows
+  set(gtest_force_shared_crt # cmake-lint: disable=C0103
+      ON
+      CACHE BOOL "" FORCE)
+
+  # store path to googletest in variable
+  set(GTEST_PATH "extern/ddsim/extern/qfr/extern/dd_package/extern/googletest")
```

Comment:
Not that the googletest path is subject to change once you rebase the QFR (now MQT Core) PR and the DSSIM/QCEC PRs. Then this becomes: `extern/ddsim/extern/mqt-core/extern/googletest`